src/validation.h

```diff
@@ -82,9 +82,9 @@ static const int MAX_SCRIPTCHECK_THREADS = 16;
 /** -par default (number of script-checking threads, 0 = auto) */
 static const int DEFAULT_SCRIPTCHECK_THREADS = 0;
 /** Number of blocks that can be requested at any given time from a single peer. */
-static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 16;
+static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 128;
```

Comment:
i tried this before, but in my case, it was not faster.

Reply:
on its own, it isnt faster - with the multiple nodes portion above, it is noticeably faster